src/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -439,7 +439,7 @@ public static Document getXMLDocument(Reader reader, CacheTable<String> stringCa
         return doc;
     }
 
-    private void parseDoc(Map<String, String> namespacePrefixesByUri) {
+    private void parseDoc(Map<String, String> namespacePrefixesByUri, String lastSavedSrc) {
```

Comment:
~Here there's no need to maintain backwards compatibility because this method was already private.~ Don't mind me ;)